flake.nix

```
@@ -32,13 +32,23 @@
           legacyPackages = devshell;
           devShell = devshell.fromTOML ./devshell.toml;
         };
+      fromTOML = path:  eachSystem (system:
+        let pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ (import ./overlay.nix) ];
+        };
+        in {
+          devShell = pkgs.devshell.fromTOML path;
+        }
+      );
     in
     {
       defaultTemplate.path = ./template;
       defaultTemplate.description = "nix flake new 'github:numtide/devshell'";
       # Import this overlay into your instance of nixpkgs
       overlay = import ./overlay.nix;
       lib = {
+        inherit fromTOML;
```

```suggestion
        inherit flakeTOML;
```
What do you think? `fromTOML` doesn't really jump to me to know what it does.